datafusion/sql/src/planner.rs

```diff
@@ -921,25 +887,31 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         plan: LogicalPlan,
         alias: TableAlias,
     ) -> Result<LogicalPlan> {
-        let columns_alias = alias.clone().columns;
-        if columns_alias.is_empty() {
-            // sqlparser-rs encodes AS t as an empty list of column alias
```

Comment:
Original code is bug (add twice) + bug (when columns-alias, forget to add alias). let me have explain.
The process is as follows
- Add table-alias (above those code are removed)
- Add table-alias and columns-alias (these code here)
  - bug: when  columns-alias is empty, these code will ignore table-alias

so we just unify them into 
-  Add table-alias and columns-alias